docs/cluster.md

```diff
@@ -5,12 +5,13 @@ The `amp cluster` command manages all cluster-related operations for AMP.
 ```
     $ amp cluster
 
-    Usage:	amp cluster [OPTIONS] COMMAND [ARGS...]
+    Usage:	amp cluster COMMAND
 
     Cluster management operations
 
     Options:
-          --help   Print usage
+      -h, --help            Print usage
+      -s, --server string   Specify server (host:port)
```

Comment:
Also, we'd like to keep a few md files from legacy, and update the README.md. Some links are broken now.